drivers/docker/driver.go

```diff
@@ -653,7 +653,7 @@ func (d *Driver) pullImage(task *drivers.TaskConfig, driverConfig *TaskConfig, r
 		AllocID:   task.AllocID,
 		TaskName:  task.Name,
 		Timestamp: time.Now(),
-		Message:   "Downloading image",
+		Message:   fmt.Sprintf("Downloading image %s", dockerImageRef(repo, tag)),
 		Annotations: map[string]string{
 			"image": dockerImageRef(repo, tag),
 		},
```

Comment:
This makes me realize that we already have all the data we need here in the annotations and it already shows up in the API:

```
$ nomad operator api "/v1/allocation/018da10f-16c2-7bba-7905-e0bea88aebc7" | jq '.TaskStates["task"].Events[2]'
{
  "Details": {
    "image": "busybox:1"
  },
  "DiskLimit": 0,
  "DisplayMessage": "Downloading image",
  "DownloadError": "",
  "DriverError": "",
  "DriverMessage": "Downloading image",
  "ExitCode": 0,
  "FailedSibling": "",
  "FailsTask": false,
  "GenericSource": "",
  "KillError": "",
  "KillReason": "",
  "KillTimeout": 0,
  "Message": "",
  "RestartReason": "",
  "SetupError": "",
  "Signal": 0,
  "StartDelay": 0,
  "TaskSignal": "",
  "TaskSignalReason": "",
  "Time": 1722525403089176300,
  "Type": "Driver",
  "ValidationError": "",
  "VaultError": ""
}
```

We're just not displaying these details in the UI or CLI. That seems like a better place to do this rather than repeating state that already exists, doesn't it?

Reply:
Yes it makes sense, in fact this DriverMessage/DisplayMessage can be used for localisation, hence displaying it only on the UI side is better approach, let me change that.